.gitlab-ci.yml

```diff
@@ -468,6 +468,8 @@ test_accep_qemux86_64_uefi_grub:
     JOB_BASE_NAME: mender_qemux86_64_uefi_grub
   after_script:
     - export WORKSPACE=$(realpath ${CI_PROJECT_DIR}/..)
+    - mkdir -p stage-artifacts
+    - cp $WORKSPACE/qemux86-64-uefi-grub/qemux86-64-uefi-grub_release_1_*.mender stage-artifacts/
```

Comment:
These names (`qemux86-64-uefi-grub`, and `vexpress-qemu` in the next section, etc) are repeated a lot. Would it make sense to provide a variable for these in this job, and then refer to that? That would make it easier to copy paste this if we need a new configuration.

It's not important to fix in this PR, just something to consider submitting separately, perhaps.

Reply:
I thought about it, but concluded that being explicit will be following the rest of the job design (we already have explicit JOB_BASE_NAME, or results xml file name, etc) and that whenever you need to copy it for a new job, you better rename all of it.

If GitLab improves the templates management in future releases, we might be able to nicely refactor all these jobs with variables. But at the moment it is not possible as you can only "template" a given task (after_script in these cases)